CryptoPrices/Sources/MyCoin/Sources/MyCoin/TimeFrameSectionView/TimeFrameSection.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  TimeFrameSection.swift
+//  MyCoin
+//
+//  Created by Khanh on 04/01/2023.
+//
+
+import Styleguide
+import SwiftUI
+
+struct TimeFrameSection: View {
+    
+    @State var selected: TimeFrameItem = .init(timeFrame: .oneDay)
+    
+    private var timeFrameList: [TimeFrameItem] = [
+        .init(timeFrame: .oneDay),
+        .init(timeFrame: .oneWeek),
+        .init(timeFrame: .oneMonth),
+        .init(timeFrame: .oneYear),
+        .init(timeFrame: .fiveYears)
+    ]
+
+    var body: some View {
+        VStack(alignment: .center) {
+            HStack {
+                ForEach(timeFrameList) { item in
+                    Button(item.title) {
+                        selected = TimeFrameItem(timeFrame: item.timeFrame)
```

Comment:
I think we might need another TODO here for handling loading chart logic 🤔

Reply:
Nice! I fixed it at https://github.com/nimblehq/nimble-crypto-ios/pull/86/commits/e096d963ba508001f5aae3513b0af1dc28571083.